Gemfile

```diff
@@ -1,9 +1,20 @@
 # foreman plugins import this file therefore __FILE__ cannot be used
 FOREMAN_GEMFILE = __FILE__ unless defined? FOREMAN_GEMFILE
 
+require_relative 'config/boot_settings'
+
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.8'
+case SETTINGS[:rails]
+when '4.2'
+  gem 'rails', '4.2.8'
+when '5.0'
+  gem 'rails', '5.0.3'
```

Comment:
@dLobatog largely correct, though Fedora and EL7 are currently packaged with, and would remain on Ruby on Rails 4.2 until it's changed.
@ohadlevy not at this time.